core/shared/src/main/scala/clue/StreamingClientStatus.scala

```diff
@@ -3,10 +3,16 @@
 
 package clue
 
+import cats.Eq
+import cats.Show
+
 sealed trait StreamingClientStatus
```

Comment:
Should this be an `Enumerated`?

Reply:
I'm not entirely convinced of introducing a dependency on `lucuma-core` for this.

Reply:
Oh, I thought `lucuma-core` was there already. I agree it is not needed in that case